packages/kas/.eslintrc.js

```diff
@@ -1,16 +1,7 @@
-/* eslint-disable @typescript-eslint/no-require-imports */
 /* eslint-disable import/no-commonjs */
-const path = require("path");
 
 module.exports = {
     rules: {
-        "import/no-extraneous-dependencies": [
-            "error",
-            {
-                packageDir: [__dirname, path.join(__dirname, "../../")],
-                includeTypes: true,
-            },
-        ],
```

Comment:
This is a core part of this PR. We can get rid of this rule in each package's `.eslintrc.js`, making our linting setup simpler (I couldn't figure out how to get rid of the circular imports rules below, so this file needs to stay here still).